### ISSUE TITLE
My profile can not be analyzed due to error

### ISSUE THREAD

**Issue by Strernd** [tanykim/beer-match#2]
I get following error, when entering my username "strernd"

Your 'max_id' is too low, please use a valid that is closer to the most recent ID. We only allow scanning back to a max of 300 checkins when browsing other people's checkins.

**Comment by tanykim** [comment]
Yes, I am aware of the issue. Untappd changed their API policy, the max number is not under my control. I have to update the site to allow users to authenticate to read their data. It's in my to-do list but I am afraid that I can't guarantee when I can work on this...
